test-network/scripts/deployCC.sh

```diff
@@ -33,51 +33,8 @@ CC_SRC_LANGUAGE=$(echo "$CC_SRC_LANGUAGE" | tr [:upper:] [:lower:])
 
 FABRIC_CFG_PATH=$PWD/../config/
 
-# User has not provided a path, therefore the CC_NAME must
```

Comment:
This may be personal preference, but I would keep the default option of using deploying the basic chaincode in go if no other default is provided. This would make basic testing easier. Users would need to supply name and path for all other chaincodes

Reply:
This makes sense to me, @denyeart thoughts on this?

Reply:
@nikhil550 The whole reason we are removing the short name support is so that users learn a single way to deploy chaincode using the test network from the beginning, so that they can easily understand how to deploy their own chaincodes. I think most users will copy/paste the entire command when running the tutorials, so it doesn't seem to save typing. And once they get familiar with the command, it is an easy mental leap to replace the name/path/language with their own chaincode. Does that make sense?

Reply:
The change makes sense. My only thought is that a default would make it easier to test a new version or code change, ie with:
```
./network.sh up createChannel
./network.sh deploy
```

and that it would not be a problem if only one path was provided as a default and users had to learn the others, but if it would only add confusion it would be better to remove. If the default is removed, this PR removes the only use of the shortnames in the samples: https://github.com/hyperledger/fabric-samples/pull/392

Reply:
@nikhil550 Thanks for fixing that last remaining default usage #392 , I've merged it.

My other concern with leaving the defaults in, is if somebody forgets to pass the name, path, or language, it will simply take the basic sample default and they may be confused when something goes wrong. If there are no defaults, then we can provide a clear message indicating which property is missing. My suggestion is to remove the defaults and make it explicit so that there is no chance for confusion.

Reply:
I was leaning towards having the default but Dave's idea sounds pretty good. In case nothing is specified the error message could literally indicate the command to type to have "the default".

Reply:
sounds good to me